src/content/fr/_shared/udacity/ud860.html

```diff
@@ -0,0 +1,28 @@
+<div itemscope itemtype="http://schema.org/Course">
+    <h2 itemprop="name">Optimisations relatives au rendu du navigateur</h2>
+    <a href="https://www.udacity.com/course/browser-rendering-optimization--ud860">
+      <img src="/web/shows/udacity/img/ud860.jpg" class="attempt-right" itemprop="image">
+    </a>
+    <p itemprop="description">
+      La performance a une réelle importance pour les utilisateurs. Les développeurs
+      web construisent des applications dont les interactions sont rapides et le rendu
+      fluide. L'expert Paul Lewis vous enseignera la manière de réduire ce jank et 
+      de créer des applications web qui sont au niveau des 60 FPS préconisés.
+      A la suite de ce cours, vous aurez tous les outils nécessaires pour correctement identifier
+      et diagnostiquer les causes de jank. Vous explorerez le pipeline de rendu du navigateur
+      ainsi que les bonnes pratiques de conception pour construire des applications performantes.
+    </p>
+    <p>
+      Il s'agit d'un cours gratuit offert par
+      <span itemprop="Provider">
+        <span itemscope itemtype="http://schema.org/Organization">
+          <a href="https://www.udacity.com" itemprop="sameAs">
+            <span itemprop="name">Udacity</span>
+          </a>
+        </span>
+      </span>
+    </p>
+    <a href="https://www.udacity.com/course/browser-rendering-optimization--ud860" class="external">Take Course</a>
```

Comment:
"Suis le cours" or "Prendre un cours"?

Reply:
I'll go for "Suivre le cours", if that's okay for you.